docs/src/getting-started/installation.md

```diff
@@ -52,6 +52,63 @@ Install the latest version:
 asdf install starknet-foundry latest
 ```
 
+### Set the installed version:
```

Comment:
I suppose this and every other paragrapgh should be a subsection of `### Installation via asdf`.

Lets use `####`

Reply:
I'd remove this paragraph entirely, this is still part of `### Installation via [asdf](https://asdf-vm.com/)`.